### ISSUE TITLE
svgo removes marker that is used by `marker-end:url(#marker-id)`

### ISSUE THREAD

**Issue by rnwst** [svg/svgo#1725]
### Describe the bug
When optimizing the below SVG, svgo removes the marker with Id `marker2`, even though it is used by the last path element's style attribute in `marker-end:url(#marker2)`.

### To Reproduce
**1. Run the following command (see below for `file.svg`):**
``` shell
svgo --config config.js file.svg --pretty --indent=2
```
`file.svg`:
``` svg
<svg xmlns="http://www.w3.org/2000/svg" width="65mm" height="56mm" viewBox="0 0 65 56">
  <defs>
    <marker id="marker1" markerHeight="5" markerWidth="8" orient="auto-start-reverse" overflow="visible">
      <path d="m0 0-2.5 2.5L6.25 0-2.5-2.5Z"/>
    </marker>
    <marker id="marker2" markerHeight="5" markerWidth="8" orient="auto-start-reverse" overflow="visible">
      <path d="m0 0-2.5 2.5L6.25 0-2.5-2.5Z"/>
    </marker>
  </defs>
  <path d="M24 21v45h51" style="fill:none;stroke:#000;stroke-width:.352778;marker-start:url(#marker1);marker-end:url(#marker2)" transform="translate(-17 -17)"/>
</svg>
```
`config.js`:
``` javascript
module.exports = {
  plugins: [
    {
      name: 'preset-default',
      params: {
        overrides: {
          cleanupIds: {
            minify: false,
          },
        },
      },
    },
  ],
};
```
The configuration needs to be done to avoid [this issue](https://github.com/svg/svgo/issues/1719).

**2. Observe that the resultant SVG is now missing an arrowhead when rendered.** This is because svgo has removed the marker with Id `marker2`.

### Expected behavior
The second marker should not be removed, since it is used in `marker-end:url(#marker2)`.

### Screenshots
Before svgo:
![image](https://user-images.githubusercontent.com/98110034/205417976-1beb1e96-a6f3-44b6-902f-05f5bae5c520.png)
After svgo:
![image](https://user-images.githubusercontent.com/98110034/205418008-7343e135-6805-417c-ad04-b8f29f947442.png)

### Desktop
 - SVGO Version 3.0.2
 - NodeJs Version 19.2.0
 - OS: Linux
 
### Additional context
SVG before svgo:
`file.svg`:
``` svg
<svg xmlns="http://www.w3.org/2000/svg" width="65mm" height="56mm" viewBox="0 0 65 56">
  <defs>
    <marker id="marker1" markerHeight="5" markerWidth="8" orient="auto-start-reverse" overflow="visible">
      <path d="m0 0-2.5 2.5L6.25 0-2.5-2.5Z"/>
    </marker>
    <marker id="marker2" markerHeight="5" markerWidth="8" orient="auto-start-reverse" overflow="visible">
      <path d="m0 0-2.5 2.5L6.25 0-2.5-2.5Z"/>
    </marker>
  </defs>
  <path d="M24 21v45h51" style="fill:none;stroke:#000;stroke-width:.352778;marker-start:url(#marker1);marker-end:url(#marker2)" transform="translate(-17 -17)"/>
</svg>
```

SVG after optimization by svgo:
``` svg
<svg xmlns="http://www.w3.org/2000/svg" width="65mm" height="56mm" viewBox="0 0 65 56">
  <defs>
    <marker id="marker1" markerHeight="5" markerWidth="8" orient="auto-start-reverse" overflow="visible">
      <path d="m0 0-2.5 2.5L6.25 0-2.5-2.5Z"/>
    </marker>
  </defs>
  <path d="M24 21v45h51" style="fill:none;stroke:#000;stroke-width:.352778;marker-start:url(#marker1);marker-end:url(#marker2)" transform="translate(-17 -17)"/>
</svg>
```

**Comment by SethFalco** [comment]
Hey! Thanks for reporting the issue!  
This has been resolved in v3.0.3 and was released last week.

**Comment by rnwst** [comment]
Thanks for your work in fixing this! I can confirm that this is fixed now in v3.0.3.
